getting-started.md

```diff
@@ -42,9 +42,20 @@ home screen named `/`, which builds a `FlutterDemo` component.
 To learn more about how to build apps with Flutter, please see the
 [tutorial](/tutorial/).
 
-Setting up your Android device
+Getting Started with Atom
```

Comment:
Conceptually this section goes after "Setting up your Android device" because it's not a required step to getting something showing on your screen.  It should go above "debugging" though.